development/orders-lifecycle/_index.md

```diff
@@ -1,82 +1,120 @@
 ---
 title: Orders lifecycle
 weight: 42
+useMermaid: true
 ---
 
-# Order creation
+# Order lifecycle: Cart, Checkout, and Order
 
-Orders can be created from a cart either in Front Office (FO) by the customer or in Back Office (
-BO) `Sell -> Orders -> "Add new order"` by the shop admin. See [Add new order page]({{< relref "../page-reference/back-office/order/add-new-order" >}}).
+{{% notice note %}}
+Orders can be created from a cart either in Front Office (FO) by the customer or in Back Office (BO) `Sell -> Orders -> "Add new order"` by the shop admin. See [Add new order page]({{< relref "../page-reference/back-office/order/add-new-order" >}}).
+{{% /notice %}}
+
+## Carts
 
-## Cart
+### Cart in Front Office (FO)
 
-In FO by default, a new empty cart is created everytime a customer signs in - this behavior can be adjusted in BO
-`Configure -> Shop parameters -> Customer settings`. If customer is not signed in (guest) - the cart is created once
+In FO by default, a new empty cart is created in database everytime a customer signs in - this behavior can be adjusted in BO
```

Comment:
```suggestion
In FO, by default, a new empty cart is created in the database every time a customer signs in - this behavior can be adjusted in BO
```

This setting you're writing about, is it `Re-display cart at login`? If so, make sure to test it on PrestaShop 8.0 and be more explicit. This setting is not about creating or not a cart. I think the setting saying "Re-display cart at login" is connected with the behavior, and we could explicitly explain this.

Reply:
ok, i will test, that's a statement that was present in the doc. thanks